src/Drupal/Commands/core/MigrateRunnerCommands.php

```diff
@@ -57,64 +45,52 @@ class MigrateRunnerCommands extends DrushCommands
-     *   Retrieve status for all migrations tagged with <info>user</info> or
-     *   <info>main_content</info>
-     * @usage migrate:status classification,article
-     *   Retrieve status for specific migrations
-     * @usage migrate:status --field=id
-     *   Retrieve a raw list of migration IDs.
-     * @usage ms --fields=id,status --format=json
-     *   Retrieve a Json serialized list of migrations, each item containing only the migration ID and its status.
-     *
-     * @aliases ms,migrate-status
-     *
-     * @topics docs:migrate
-     *
-     * @validate-module-enabled migrate
-     *
-     * @field-labels
-     *   id: Migration ID
-     *   status: Status
-     *   total: Total
-     *   imported: Imported
-     *   needing_update: Needing update
-     *   unprocessed: Unprocessed
-     *   last_imported: Last Imported
-     * @default-fields id,status,total,imported,unprocessed,last_imported
-     * @filter-default-field status
-     * @return RowsOfFields
-     *   Migrations status formatted as table.
-     * @version 10.4
-     *
      * @throws \Exception
      *   If --names-only is used with --field having other value than 'id'.
      */
+    #[CLI\Command(name: 'migrate:status', aliases: ['ms', 'migrate-status'])]
+    #[CLI\Argument(name: 'migrationIds', description: 'Restrict to a comma-separated list of migrations. Optional.')]
+    #[CLI\Option(name: 'tag', description: 'A comma-separated list of migration tags to list. If only <info>--tag</info> is provided, all tagged migrations will be listed, grouped by tags.')]
+    #[CLI\Option(name: 'names-only', description: '[Deprecated, use --field=id instead] Only return names, not all the details (faster).')]
+    #[CLI\Usage(name: 'migrate:status', description: 'Retrieve status for all migrations')]
+    #[CLI\Usage(name: 'migrate:status --tag', description: 'Retrieve status for all migrations, grouped by tag')]
+    #[CLI\Usage(name: 'migrate:status --tag=user,main_content', description: 'Retrieve status for all migrations tagged with <info>user</info> or <info>main_content</info>')]
+    #[CLI\Usage(name: 'migrate:status classification,article', description: 'Retrieve status for specific migrations')]
+    #[CLI\Usage(name: 'migrate:status --field=id', description: 'Retrieve a raw list of migration IDs.')]
+    #[CLI\Usage(name: 'ms --fields=id,status --format=json', description: 'Retrieve a Json serialized list of migrations, each item containing only the migration ID and its status.')]
+    #[CLI\Topics(topics: ['docs:migrate'])]
+    #[CLI\ValidateModulesEnabled(modules: ['migrate'])]
+    #[CLI\FieldLabels(labels: [
+        'id' => 'Migration ID',
+        'status' => 'Status',
+        'total' => 'Total',
+        'imported' => 'Imported',
+        'needing_update' => 'Needing update',
+        'unprocessed' => 'Unprocessed',
+        'last_imported' => 'Last Imported',
+    ])]
+    #[CLI\DefaultFields(fields: [
```

Comment:
```suggestion
    #[CLI\DefaultTableFields(fields: [
```
We are switching to DefaultTableFields everywhere. DefaultTableFields only changes the table view whereas other output formats will show all the fields by default.